src/Tochka.JsonRpc.Client/Models/ISingleJsonRpcResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Text.Json;
+using JetBrains.Annotations;
+using Tochka.JsonRpc.Common.Models.Response.Errors;
+
+namespace Tochka.JsonRpc.Client.Models;
+
+[PublicAPI]
+public interface ISingleJsonRpcResult
```

Comment:
doc?

Reply:
will be added later with all other documentation